ConwaysGameOfLife/Validator.cs

```diff
@@ -5,14 +5,45 @@ namespace ConwaysGameOfLife
     public class Validator
     {
         // validates input
-        public bool IsUniverse(Universe input)
+
+        public static bool ParseInput(string numberInput)
+        {
+            if (Int32.TryParse(numberInput, out var number))
+            {
+                return true;
+            }
+            return false;
+        }
+        
+        public bool IsValidUniverse(string input)
+        {
+            var isNumber = SplitInput(input, ",");
+            if (!input.Contains(",")) return false;
+            if (input.Length > 3) return false;
+            if (isNumber.Length != 2) return false;
+            if (!ParseInput(isNumber[0]) || !ParseInput(isNumber[1])) return false;
+            if (Int32.Parse(isNumber[0]) <= 0) return false;
+            return true;
+        }
+
+        public bool IsValidLocation(string input, int gridWidth, int gridLength)
         {
-            return input.GridWidth > 0 && input.GridLength > 0;
+            var isNumber = SplitInput(input, ","); 
+            if (!input.Contains(",")) return false;
+            if (input.Length > 3) return false;
+            if (isNumber.Length != 2) return false;
+            if (!ParseInput(isNumber[0])) return false;
+            if (!ParseInput(isNumber[1])) return false;
+            if (Int32.Parse(isNumber[0]) < 0) return false;
+            if (Int32.Parse(isNumber[1]) < 0) return false;
+            if (Int32.Parse(isNumber[0]) >= gridWidth) return false;
+            if (Int32.Parse(isNumber[1]) >= gridLength) return false;
+            return true;

```

Comment:
same here with the ordering and `char ','`.

As for combining these into a nice boolean, we could use some intermediate variables e.g. (not 100% on syntax as just using the github editor)
```
            var containsComma = input.Contains(",");

            var components = SplitInput(input, ",");
            var hasTwoComponents = components.Length == 2;
            var xIsValid = Int32.TryParse(components[0], out var x) && x >= 0 && x < gridWidth;
            var yIsValid = Int32.TryParse(components[1], out var y) && y >= 0 && y < gridLength;

            return containsComma && hasTwoComponents && xIsValid && yIsValid;
```